### ISSUE TITLE
Invalid no longer built dependencies still referenced in dependencyManagement

### ISSUE THREAD

**Issue by paulgallagher75** [vert-x3/vertx-dependencies#51]
the dependencyManagement of vertx-dependencies has a reference to vertx-mail and vertx-mongo however that is not built and released since 3.5.1 almost 3 years ago
```
      <dependency>
        <groupId>io.vertx</groupId>
        <artifactId>vertx-mail</artifactId>
        <version>4.0.0.CR1</version>
      </dependency>
      <dependency>
        <groupId>io.vertx</groupId>
        <artifactId>vertx-mongo</artifactId>
        <version>4.0.0.CR1</version>
      </dependency>
```

vertx-zookeeper was also in vertx-dependencies, but was not tagged and released for 4.0.0.CR1, if it has been dropped it should be removed from the here 
```
      <dependency>
        <groupId>io.vertx</groupId>
        <artifactId>vertx-zookeeper</artifactId>
        <version>4.0.0.CR1</version>
      </dependency>
```

**Comment by paulgallagher75** [comment]
Also vertx-mongo has not built and released since 3.5.1
```
      <dependency>
        <groupId>io.vertx</groupId>
        <artifactId>vertx-mongo</artifactId>
        <version>4.0.0.CR1</version>
      </dependency>
```

**Comment by tsegismont** [comment]
@paulgallagher75 `vertx-mail` and `vertx-mongo` can be removed. These were the parent modules of the mail and Mongo clients but they no longer exists.

`vertx-zookeeper` is just late to update to the latest clustering SPI but it's still active
